share/completions/unzip.fish

```diff
@@ -23,6 +23,13 @@ complete -c unzip -s X -d "restore UID/GID info"
 complete -c unzip -s V -d "retain VMS version numbers"
 complete -c unzip -s K -d "keep setuid/setgid/tacky permissions"
 complete -c unzip -s M -d "pipe through `more` pager"
+# Some distro has -O and -I, some hasn't.
+if unzip --help | grep -q "\-O"
```

Comment:
First of all: `\-` doesn't do what you want. You want `--`.

Secondly: Make this `string match -rq` to avoid the grep dependency:

```fish
if unzip --help | string match -rq -- -O
```